apps/explorer/src/pages/home/Home.tsx

```diff
@@ -33,7 +30,18 @@ function HomeStatic() {
                 <LatestTxCard />
             </section>
             <section className="right-item">
-                <TopValidatorsCardStatic />
+                <div data-testid="validators-table">
```

Comment:
I don't love this duplication but I think it's fine given we want to get rid of the static mode anyway

Reply:
I will remove the `HomeStatic` component.

Reply:
If you can, great. Not sure if there's still static tests that depend on it.